editoast/editoast_derive/src/lib.rs

```diff
@@ -237,3 +237,37 @@ pub fn model(input: proc_macro::TokenStream) -> proc_macro::TokenStream {
         .unwrap_or_else(darling::Error::write_errors)
         .into()
 }
+
+#[cfg(test)]
+pub(crate) fn pretty_tokens(tokens: &proc_macro2::TokenStream) -> String {
```

Comment:
Not sure that it matters to have `pub(crate)` instead of `pub` since it's protected by `#[cfg(test)]` anyway 🤷

Reply:
Done in 61411e63a

Reply:
Actually, cf. 227047406

I have to `pub(crate) use assert_macro_expansion;` otherwise Rust complains about a `missing #[macro_export]`. I figured it would be more consistent to keep both definitions visible at `pub(crate)` (even if you're right, it's not different from `pub` in this context). I'll let you resolve if you agree, I don't mind changing it either :)

Reply:
I should have added `nitpick` to my comment 😅 sure it's fine